huaweicloud/services/bms/resource_huaweicloud_bms_instance.go

```diff
@@ -240,7 +240,12 @@ func ResourceBmsInstance() *schema.Resource {
 			"agency_name": {
 				Type:     schema.TypeString,
 				Optional: true,
-				ForceNew: true,
+				Computed: true,
+			},
+			"metadata": {
```

Comment:
Please explain why metadata is not backfilled during read.
I think it's better to add a comment.